variables.tf

```diff
@@ -148,4 +148,11 @@ variable "ubuntu_ami" {
   }
 }
 
+variable "datadog_api_key" {
+  description = "Datadog api key"
+}
 
+variable "env" {
+  description = "Environment"
+  default     = "deploy-dev"
```

Comment:
worthwhile not having this default value in order to make sure we don't try and put our platform related elements in the forked repo